spec/std/concurrent_spec.cr

```diff
@@ -41,4 +41,38 @@ describe "concurrent" do
     spawn method_named("foo"), name: "foo"
     Fiber.yield
   end
+
+  it "runs something every X time" do
+    counter = 0
+
+    spawn do
+      every(10.milliseconds) do |i|
```

Comment:
Why? Even 1ms is aeons at computer speeds. if it turns out to be a problem it can probably be increased.